mm2src/coins/lightning/ln_sql.rs

```diff
@@ -1189,7 +769,7 @@ impl DbStorage for LightningPersister {
     }
 
     async fn add_or_update_payment_in_db(&self, info: PaymentInfo) -> Result<(), Self::Error> {
-        let for_coin = self.storage_ticker.clone();
+        let for_coin = self.db_ticker.clone();
         let payment_hash = hex::encode(info.payment_hash.0);
         let (is_outbound, destination) = match info.payment_type {
             PaymentType::OutboundPayment { destination } => (true as i32, Some(destination.to_string())),
```

Comment:
As I can see, `ToSql` is implemented for `bool`, so it's not required to cast it to i32.